assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -522,15 +534,6 @@ const baseActions = {
 			return;
 		}
 
-		if ( ! measurementID ) {
```

Comment:
This was a bit of a mistake that we would only set these in ! case if the user had the GTM scope. Without this change though it resulted in the Enhanced Measurement switch being stuck in a loading state when selecting an account that didn't have a matching property.